Dockerfile

```diff
@@ -5,9 +5,10 @@ RUN rustup target add x86_64-unknown-linux-musl
 RUN USER=root cargo new dotenv-linter
 WORKDIR /usr/src/dotenv-linter
 COPY Cargo.toml ./
-RUN cargo build --release
-
 COPY src ./src
+COPY benches ./benches
```

Comment:
Do we need benchmarks in a docker container? 🤔

Reply:
Yes, otherwise we get an error when building an image:
https://github.com/dotenv-linter/dotenv-linter/runs/2096601021?check_suite_focus=true